x-pack/plugins/maps/public/classes/layers/ems_vector_tile_layer/ems_vector_tile_layer.ts

```diff
@@ -40,26 +40,31 @@ interface SourceRequestData {
 }
 
 export class EmsVectorTileLayer extends AbstractLayer {
-  static createDescriptor(options: Partial<LayerDescriptor>) {
-    const tileLayerDescriptor = super.createDescriptor(options);
-    tileLayerDescriptor.type = LAYER_TYPE.EMS_VECTOR_TILE;
-    tileLayerDescriptor.alpha = _.get(options, 'alpha', 1);
-    tileLayerDescriptor.locale = _.get(options, 'locale', AUTOSELECT_EMS_LOCALE);
-    tileLayerDescriptor.style = { type: LAYER_STYLE_TYPE.TILE };
-    return tileLayerDescriptor;
-  }
+  private readonly _style: EMSVectorTileStyle;
 
-  private readonly _style: TileStyle;
+  static createDescriptor(options: Partial<EMSVectorTileLayerDescriptor>) {
+    const emsVectorTileLayerDescriptor = super.createDescriptor(options);
+    emsVectorTileLayerDescriptor.type = LAYER_TYPE.EMS_VECTOR_TILE;
+    emsVectorTileLayerDescriptor.alpha = _.get(options, 'alpha', 1);
+    emsVectorTileLayerDescriptor.locale = _.get(options, 'locale', AUTOSELECT_EMS_LOCALE);
+    emsVectorTileLayerDescriptor.style = { type: LAYER_STYLE_TYPE.EMS_VECTOR_TILE };
```

Comment:
EMSVectorTileStyleDescriptor defines `color` as required but its not set here. Is `color` key required? Should a default value be added? In EMSVectorTileStyle, color is given a value of empty string when no value exists.

Reply:
Thanks for catching this. `emsVectorTileLayerDescriptor.style` should call `EMSVectorTileStyle.createDescriptor` similar to how `HeatMap` layer works.